framework/src/utils/MemoryUtils.C

```diff
@@ -143,10 +164,12 @@ getMemoryStats(Stats & stats)
   stats._physical_memory = val[index_resident_size];
 
   // virtual mem
-  stats._virtual_memory = val[index_virtual_size];
+  stats._virtual_memory = 100; // val[index_virtual_size];
 
   // page faults
-  stats._page_faults = val[index_page_faults];
+  stats._page_faults = 100; // val[index_page_faults];
```

Comment:
The 3 hunks above look like leftovers from some debugging.

Was this supposed to be back to `sysconf(_SC_PAGE_SIZE)`, `val[index_virtual_size]` and `val[index_page_faults]`? If not, then delete the commented out code and will need to explain those magic numbers.